recipes/pan-plaster/meta.yaml

```diff
@@ -24,7 +24,7 @@ test:
     - example/
     - example/*
   commands:
-    - plaster $PREFIX/bin/example/*.fasta
```

Comment:
Keep the more elaborate test, but just reference the source files for the example data.
You might also want to validate the output data using grep.

Reply:
I've moved the more elaborate test to `run_test.sh`